src/pytest_workflow/plugin.py

```diff
@@ -481,7 +490,18 @@ def runtest(self):
         assert self.workflow.exit_code == self.desired_exit_code
 
     def repr_failure(self, excinfo, style=None):
-        message = (f"'{self.workflow.name}' exited with exit code " +
-                   f"'{self.workflow.exit_code}' instead of "
-                   f"'{self.desired_exit_code}'.")
+        standerr = self.workflow.stderr_file
+        standout = self.workflow.stdout_file
+        with open(standout, "rb") as standout_file, \
+             open(standerr, "rb") as standerr_file:
+            if os.path.getsize(standerr) >= self.stderr_bytes:
+                standerr_file.seek(-self.stderr_bytes, os.SEEK_END)
```

Comment:
@lucasvanb Coverage analysis shows that this line was not run.

Can you add a small test where the stderr and stdout are actually trunctated. (For example by using stderr_bytes = 1)
Youu can add a new test (new def) in fail_messages.py.